frontend/src/components/QrCode/FigureQrCode.vue

```diff
@@ -37,6 +42,13 @@ export default {
       }
     }
   },
+  methods: {
+    downloadImg() {
+      const canvas = document.getElementById('qrcanvas')
+      const image = canvas.toDataURL('image/png')
+      document.getElementById('download').href = image
+    },
+  },
```

Comment:
What about testing this method?

![qr_tests](https://user-images.githubusercontent.com/3883288/229071753-50d3c1b5-6722-4952-a93a-ddb814e4cfe4.jpg)

Reply:
Resolved by last two commits.